src/plugins/vis_type_metric/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "visTypeMetrics",
```

Comment:
should be `visTypeMetric` (without the s)

Reply:
thx! did that